lxdock/conf/schema.py

```diff
@@ -12,6 +12,7 @@
     'lxc_config': {Extra: str},
     'mode': In(['local', 'pull', ]),
     'privileged': bool,
+    'profiles': [str, ],
```

Comment:
They cannot contain spaces too. :wink: 

```
$ lxc profile create "foo bar"
$ lxc profile list
default
docker
foo
```

This is why I would use the `LXDIdentifier`, which is restrictive enough to ensure consistent profile names (but perhaps too restrictive). Otherwise you could add a `ProfileName` validator.